spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java

```diff
@@ -171,6 +180,86 @@ public void setMaxUriTags(int maxUriTags) {
 
 		}
 
+		public static class Request {
+
+			public Request(String metricName) {
```

Comment:
With 2.2 this will break as the binding will consider you want to use constructor binding with a single property. Please remove that constructor.